recipe.html

```diff
@@ -1,226 +1,303 @@
+            aria-controls="navbarNavDropdown"
+            aria-expanded="false"
+            aria-label="Toggle navigation"
+          >
+            <span class="navbar-toggler-icon"></span>
+          </button>
+          <div class="collapse navbar-collapse" id="navbarNavDropdown">
+            <ul class="navbar-nav">
+              <li class="nav-item dropdown">
+                <a
+                  class="nav-link dropdown-toggle"
+                  href="#"
+                  role="button"
+                  data-bs-toggle="dropdown"
+                  aria-expanded="false"
+                >
+                  Breakfast
+                </a>
+                <ul class="dropdown-menu">
+                  <li><a class="dropdown-item" href="#">Porridge</a></li>
+                  <li><a class="dropdown-item" href="#">Pancakes</a></li>
+                  <li><a class="dropdown-item" href="#">Breakfast Wrap</a></li>
+                </ul>
+              </li>
+              <li class="nav-item dropdown">
+                <a
+                  class="nav-link dropdown-toggle"
+                  href="#"
+                  role="button"
+                  data-bs-toggle="dropdown"
+                  aria-expanded="false"
+                >
+                  Main dishes
+                </a>
+                <ul class="dropdown-menu">
+                  <li><a class="dropdown-item" href="#">Lasagna</a></li>
+                  <li><a class="dropdown-item" href="#">Main dish 2</a></li>
+                  <li><a class="dropdown-item" href="#">Main dish 3</a></li>
+                </ul>
+              </li>
+              <li class="nav-item dropdown">
+                <a
+                  class="nav-link dropdown-toggle"
+                  href="#"
+                  role="button"
+                  data-bs-toggle="dropdown"
+                  aria-expanded="false"
+                >
+                  Dessert
+                </a>
+                <ul class="dropdown-menu">
+                  <li><a class="dropdown-item" href="#">Brownies</a></li>
+                  <li><a class="dropdown-item" href="#">Rice pudding</a></li>
+                  <li><a class="dropdown-item" href="#">Apple crumble</a></li>
+                </ul>
+              </li>
+              <li class="nav-item dropdown">
+                <a
+                  class="nav-link dropdown-toggle"
+                  href="#"
+                  role="button"
+                  data-bs-toggle="dropdown"
+                  aria-expanded="false"
+                >
+                  Drinks
+                </a>
+                <ul class="dropdown-menu">
+                  <li><a class="dropdown-item" href="#">Chai Latte</a></li>
+                  <li><a class="dropdown-item" href="#">Mango Lassi</a></li>
+                  <li><a class="dropdown-item" href="#">Moscow Mule</a></li>
+                </ul>
+              </li>
+            </ul>
+          </div>
+        </div>
```

Comment:
i think this </div> has to be deleted. 
not sure right now, maybe we simply check it when we merged and pulled :)